exist-core/src/main/java/org/exist/xquery/functions/fn/ExtCollection.java

```diff
@@ -41,35 +40,33 @@
 
 import java.net.URI;
 import java.net.URISyntaxException;
-import java.util.ArrayList;
 import java.util.Iterator;
-import java.util.List;
+
+import static org.exist.xquery.FunctionDSL.*;
 
 /**
- * @author wolf
+ * @author <a href="mailto:adam@evolvedbinary.com">Adam Retter</a>
```

Comment:
The author is still in here, from the discussion in this thread they should all be removed.